### DISCUSSION TITLE
Avast detection

### DISCUSSION THREAD

**Discussion by CipherSniff** [Atlas-OS/Atlas#1182] [Help & Support] [answered]
![20240813_003708.jpg](https://github.com/user-attachments/assets/7c13c88f-d1a5-4db1-ae55-fa2914319507)

**Comment by trolljoe** [comment]
not an atlas issue, you had avast installed (which I recommend you to uninstall and just use defender)

**Comment by Ast3risk-ops** [comment, marked as answer]
Avast's boot-time scan seems to have falsely identified Atlas components as malicious, just ignore it.
